### DISCUSSION TITLE
How are Angular version updates handled?

### DISCUSSION THREAD

**Discussion by phihochzwei** [nrwl/nx#19917] [General]
There is this one question that bothers me: How am I supposed to handle the used Angular Version for my project? Usually I use nx migrate latest to keep everything up to date, but I see that this keeps me always behind the angular releases. E.g. I have NX 17.0.0 in place which uses NG 16.2.0, but the currently latest ist NG 16.2.10 

Is there a rule of thumb how often this gets updated in NX?

Sorry if this has been answered before, but I could not find anything about this.

**Comment by fen89** [comment]
Depends on the angular release but the NX team catches up asap imho on new releases. 

You can always check the version matrix here: https://nx.dev/nx-api/angular/documents/angular-nx-version-matrix
